src/components/selectionBoxLayer.ts

```diff
@@ -14,6 +14,8 @@ export module Component {
     constructor() {
       super();
       this.classed("selection-box-layer", true);
+      this._fixedWidthFlag = true;
+      this._fixedHeightFlag = true;
```

Comment:
Zero. Really `_fixedWidthFlag` should be named "Would use extra offered width" or something.